lib/assigner/include/zkevm_word.hpp

```diff
@@ -183,6 +189,11 @@ namespace nil {
                 return static_cast<uint64_t>(value[i]);
             }
 
+            const value_type &to_value_type() const
```

Comment:
Why we need this conversion?
If only for test `to_string` - may be better enable `to_string` method

Reply:
Actually there is one place in `set_val` test where it used outside `to_string()`:
https://github.com/NilFoundation/evm-assigner/blob/d165a1086e56fac60888e9a39a320adf5187f8c1/lib/assigner/test/assigner_test.cpp#L313
It can be replaced with `to_string()` too, but I just thought that it can be useful to have a conversion into `intx::uint256` since we are not going to abandon it.

Reply:
ok, let's just rename it to `get()` - common approach (smart pointers, optional, ...)

Reply:
Done